ui-tests/tst/software/aws/toolkits/jetbrains/uitests/fixtures/IdeaFrame.kt

```diff
@@ -92,10 +92,10 @@ class IdeaFrame(remoteRobot: RemoteRobot, remoteComponent: RemoteComponent) : Co
         }
     }
 
-    // Tips sometimes open when running locally, close it if it opens
+    // Tips sometimes open when running, close it if it opens
     fun tryCloseTips() {
```

Comment:
Can we kill this now?

Reply:
Nope, these show up on the tests running on codebuild

Reply:
Oh right...d'oh thats 203..